roadie-kubernetes/templates/cluster-role.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.clusterRole.name }}
+rules:
+  {{- toYaml .Values.clusterRole.rules.set1 | nindent 2 }}
+  {{- toYaml .Values.clusterRole.rules.set2 | nindent 2 }}
+  {{- toYaml .Values.clusterRole.rules.set3 | nindent 2 }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
should this be in a different file.. not sure. just asking.

Reply:
Doesn't matter - its related to the cluster role. If you like I can